tests/integration/ardrive.int.test.ts

```diff
@@ -112,21 +112,57 @@ describe('ArDrive class - integrated', () => {
 
 	describe('folder function', () => {
 		describe('createPublicFolder', () => {
+			beforeEach(() => {
+				stub(arfsDao, 'getPublicChildNamesOfParentFolderId').resolves(['CONFLICTING_NAME']);
+			});
+
+			it('throws an error if the folder name conflicts with another folder in the parent', async () => {
```

Comment:
I find `in the parent` to be ambiguous. "The parent" could refer to two different things: the origin parent or destination parent. I think it's easy enough to say "the destination folder".